src/shivani/Assignment12/Assingment_12.java

```diff
@@ -0,0 +1,44 @@
+package shivani.Assignment12;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+import base.PredefinedActions;
+
+/*program 1: count number of employee under each manager, verify count against properties file.
+output : 20209 -> 3
+         20205 -> 4
+		 10101 -> 2*/
+public class Assingment_12 {
+	
+	static HashMap<String,Integer> getNumberOfEmpUnderEachManager(WebDriver driver) {
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method